src/pykeen/models/unimodal/complex.py

```diff
@@ -76,8 +75,10 @@ def __init__(
         regularizer: Optional[Regularizer] = None,
         preferred_device: DeviceHint = None,
         random_seed: Optional[int] = None,
-        entity_initializer=nn.init.normal_,
-        relation_initializer=nn.init.normal_,
+        # initialize with entity and relation embeddings with standard normal distribution, cf.
+        # https://github.com/ttrouill/complex/blob/dc4eb93408d9a5288c986695b58488ac80b1cc17/efe/models.py#L481-L487
+        entity_initializer: Hint[Initializer] = 'normal',
```

Comment:
Should we use the string as default value? This might make it harder to dig to the actual implementation.

Reply:
Options for defaults:
1. set string that gets looked up
2. set `None` and have class variable containing the actual value
3. put actual function in signature

The issue with 3) might be in optuna or automatic CLI generation. I guess we could use a reverse map to look up the string name for the function, though